libraries/state_history/include/eosio/state_history/rocksdb_receiver.hpp

```diff
@@ -162,7 +162,7 @@ class rocksdb_receiver_single_entry : public rocksdb_receiver {
       table_ = {};
    }
 
-   void set_delta_present_flag(bool value) {
+   void set_delta_present_flag(uint8_t value) {
```

Comment:
Now that present is not necessarily a bool, the "flag" in the name suffix doesn't seem quite correct, could you please change it to just set_delta_present ?

Reply:
Done